test/libsolidity/semanticTests/storageLayoutSpecifier/variable_cleanup.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+    uint8 x;
+    int8 y;
+    bytes2 b;
+    // Note the return types are larger than the state variable ones
+    function f(uint _x, int _y, bytes3 _b) public returns (uint16, int32, bytes32) {
+        x = uint8(_x);
+        y = int8(_y);
+        b = bytes2(_b);
+        require(b == "ab");
+        return (x, y, b);
+    }
+}
+// ----
+// f(uint256,int256,bytes3): 256, -129, "abc" -> 0, 127, 0x6162000000000000000000000000000000000000000000000000000000000000
```

Comment:
This is another case where formatting the expectations manually makes it much quicker to verify their correctness at a glance:. You can see the truncation immediately if you use the right types of literals and keep them the same on both sides:

```suggestion
// f(uint256,int256,bytes3): 0x0100, 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7f, "abc" -> 0x00, 0x7f, "ab"
```

Reply:
Fixed.